capstone/src/main/java/com/google/sps/servlets/Constants.java

```diff
@@ -33,4 +36,8 @@ private Constants() {}
   static final String NEW_YEAR_PROP = "new-year";
   static final String NEW_MAJOR_PROP = "new-major";
   static final String PROFILE_PIC_PROP = "upload-profile";
+  static final String CALENDAR_PROP = "calendar";
+  public static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+  public static final String GOOGLE_APPLICATION_NAME = "google.com:clubhub-step-2020";
+  public static final String PST_TIMEZONE = "America/Los_Angeles";
```

Comment:
I think Kevin just merged his PR where he adds a constant called "TIMEZONE" for "America/Los_Angeles" as well! Please do resolve that when merging this!